assets/scss/header.scss

```diff
@@ -311,4 +311,40 @@
   }
 }
 
+
+.nhsuk-header__search--results-page {
+	float: unset;
+	margin-left: 0;
+
+	& .nhsuk-search__input {
+		float: none;
+		width: calc(100% - 54px);
+		display: inline-block;
+	}
+
+	& .nhsuk-search__submit {
+		float: none !important;
```

Comment:
Just double checking theres no way we can use class names (or ids) rather then important

Reply:
Yes using important is a bit of a last resort. I went with it to get something done quick, and to override something else that was quite high specificity, but I'll take another look at removing it with the next round of work on this where I focus more on the styling :)